File: find word containing char.java
class Solution {
    public List<Integer> findWordsContaining(String[] words, char x) {
        List<Integer> char_index = new ArrayList<>(); 
        for (int i = 0 ; i< words.length ; i++) {
            if ( words[i].indexOf(x) != -1){
                char_index.add(i);
            }
        }
        return char_index;
    }
}

File: find word containing char.py
from typing import List
class Solution:
    def findWordsContaining(self, words: List[str], x: str) -> List[int]:
        res = []
        for i , w in enumerate(words):
            if w.count(x):
                res.append(i)
        return res

s=Solution()
# Example usage:
words = ["hello", "world", "python", "programming"] 
x = "o"
print(s.findWordsContaining(words, x))  # Output: [0, 1]
